Source/WebCore/platform/graphics/gtk/IconGtk.cpp

```diff
@@ -44,12 +44,12 @@ Icon::~Icon()
 {
 }
 
-RefPtr<Icon> Icon::create(GIcon* icon)
+RefPtr<Icon> Icon::create(GRefPtr<GIcon>&& icon)
 {
     if (!icon)
         return nullptr;
 
-    return adoptRef(new Icon(icon));
+    return adoptRef(new Icon(icon.get()));
```

Comment:
You should change the constructor too, to receive a GRefPtr&& and here WTFMove(icon) instead of passing the raw pointer